src/ClusterManager/pod_template.py

```diff
@@ -93,8 +93,6 @@ def generate_params(self, job):
 
         if "nodeSelector" not in params:
             params["nodeSelector"] = {}
-        if "gpuType" in params:
-            params["nodeSelector"]["gpuType"] = params["gpuType"]
         if "sku" in params:
```

Comment:
I also see seom gpuType keywords in framework.py, maybe delete them as well.

Reply:
Those are in `labels`. Let's do this in another PR.